frontend/src/pages/NewRoutePage.js

```diff
@@ -30,10 +30,10 @@ export default function NewRoutePage() {
 
 
     return (
-        <React.Fragment>
+        <>
             <MapContainer markers={markers} setMarkers={setMarkers} className="mapContainer"/>
             <AddNewRouteForm onSubmit={handleSubmit} routeToAdd={routeToAdd} handleChange={handleChange}/>
             <div>{markers.map(marker => <><p>{marker.lat}</p><p>{marker.locationName}</p></>)}</div>
```

Comment:
guess you get a console warning here, because you map over an array without assigning a key value. You would need to change the fragments to <React.Fragment key = _some-unique-id_>

Reply:
Gonna change that ! Thank you !